iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/algorithm.inl

```diff
@@ -71,6 +71,7 @@ inline constexpr bool doesContainType() noexcept
     return doesContainType<T, CompareType>() || doesContainType<T, Next, Remainder...>();
 }
 
+// AXIVION Next Construct AutosarC++19_03-A0.1.4 : function finalizes compile time recursion of doesContainValue
 template <typename T>
 inline constexpr bool doesContainValue(const T v IOX_MAYBE_UNUSED) noexcept
```

Comment:
Why not?

```suggestion
inline constexpr bool doesContainValue(const T) noexcept
```

Reply:
Seems like the variable name was introduced to fix a warning, see [here](https://github.com/eclipse-iceoryx/iceoryx/commit/928d02660cee561c4096d6bf461e4cc6be98c5c2) :thinking:  But let's try it again.

Reply:
I thought so too, but the clang tidy check doesn't complain.

Reply:
@FerdinandSpitzschnueffler That means we can remove the variable?

Reply:
@mossmaurice So it seems.